sunpy/data/sample.py

```diff
@@ -3,19 +3,16 @@
 from __future__ import absolute_import
 
 import sys
-from ._sample import sample_files as _sample_files
-import os.path
-
-from sunpy import config as _config
-_sampledata_dir = _config.get("downloads", "sample_dir")
+from ._sample import files as _sample_files
+from ._sample import base_urls as _base_urls
+from ._sample import get_sample_file
```

Comment:
wouldn't be eassier to keep the names as `_sample_files`, `_base_urls` in the `_sample` file? then all could go in the same line:
```python
from ._sample import _sample_files, _base_urls, get_sample_file
```